controllers/user.go

```diff
@@ -313,6 +314,8 @@ func (c *ApiController) UpdateUser() {
 	var user object.User
 	err := json.Unmarshal(c.Ctx.Input.RequestBody, &user)
 	if err != nil {
+		logger.Error(goCtx, "failed to unmarshall user",
```

Comment:
does this message have operation(`UpdateUser()`) name added under the hood? 
If not, we can change this to 

`logger.Error(goCtx, errors.Wrap(err, "UpdateUser failed: failed to unmarshal request body").Error())`

Reply:
done